doc/classes/@GlobalScope.xml

```diff
@@ -439,16 +439,18 @@
 			</description>
 		</method>
 		<method name="lerp">
-			<return type="float" />
-			<argument index="0" name="from" type="float" />
-			<argument index="1" name="to" type="float" />
-			<argument index="2" name="weight" type="float" />
+			<return type="Variant" />
+			<argument index="0" name="from" type="Variant" />
+			<argument index="1" name="to" type="Variant" />
+			<argument index="2" name="weight" type="Variant" />
```

Comment:
Why is the weight a Variant? I'm pretty sure it should only ever be a float (double).

Reply:
The function is defined correctly, so not sure.

Reply:
It's because it's binded with `FUNCBINDVR3`:
https://github.com/godotengine/godot/blob/22bb15cde52b2b0623907c889be2c71227d0258e/core/variant/variant_utility.cpp#L1414
https://github.com/godotengine/godot/blob/22bb15cde52b2b0623907c889be2c71227d0258e/core/variant/variant_utility.cpp#L1117
https://github.com/godotengine/godot/blob/22bb15cde52b2b0623907c889be2c71227d0258e/core/variant/variant_utility.cpp#L1137-L1139

Reply:
So what should it use? Doesn't look like there is a proper macro for that.

Reply:
> So what should it use? Doesn't look like there is a proper macro for that.

Yeah, there's no fitting macro. Not sure what's preferred solution but adding a new dedicated macro would work. The same as `FUNCBINDVR3` but with `get_argument_type` changed to something like `return (p_arg < 2) ? Variant::NIL : get_arg_type_helperr(VariantUtilityFunctions::m_func, p_arg); `.